pkg/operator/operator.go

```diff
@@ -340,6 +340,16 @@ func New(
 	})
 	o.informers = append(o.informers, informer)
 
+	informer = cache.NewSharedIndexInformer(
+		o.client.ClusterOperatorListWatch(ctx, "ingress"),
+		&configv1.ClusterOperator{}, resyncPeriod, cache.Indexers{},
+	)
+
+	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
Maybe add a comment that we only watch for add events because the ingress cluster operator can only from disabled to enabled?